website/content/docs/installing/no-gen-resources.mdx

```diff
@@ -0,0 +1,484 @@
+should use your cloud provider's KMS such as [AWS KMS](https://www.boundaryproject.io/docs/configuration/kms/awskms) to manage the keys Boundary
+users to encrypted sensitive information.
+
+The KMS block we're focused on is the `recovery` block. This block specifies the key used to "recover" Boundary
+but you can also use it to authenticate to Boundary to manage it as a "global" super user. This allows
+you to authenticate from the CLI or from Terraform in order to manage Boundary without any generated 
+resources. 
+
+To authenticate to Boundary using the recovery KMS workflow:
+
+<Tabs>
+<Tab heading="CLI">
+
+To use the recovery workflow on the CLI, you must pass the `-recovery-config` flag or set the environment
+variable for `BOUNDARY_RECOVERY_CONFIG` for every command ran. Authentication takes place for every command
+ran when using the recovery workflow, there is no `boundary authenticate` step:
+
+```bash
+$ export BOUNDARY_RECOVERY_CONFIG=<<EOT
+kms "aead" {
+	purpose   = "recovery"
+	aead_type = "aes-gcm"
+	key       = "8fZBjCUfN0TzjEGLQldGY4+iE9AkOvCfjh7+p0GtRBQ="
+	key_id    = "global_recovery"
+}
+EOT
+
+$ boundary connect ssh -target-id ttcp_873964g581
+...
+```
+
+</Tab>
+<Tab heading="Terraform">
+
+To configure your provider to use the recovery KMS workflow, provide the KMS block as the value for
+`recovery_kms_hcl`:
+
+```hcl
+provider "boundary" {
+  addr             = 'https://boundary.mycorp.com:9200' 
+  recovery_kms_hcl = <<EOT
+kms "aead" {
+	purpose   = "recovery"
+	aead_type = "aes-gcm"
+	key       = "8fZBjCUfN0TzjEGLQldGY4+iE9AkOvCfjh7+p0GtRBQ="
+	key_id    = "global_recovery"
+}
+EOT
+}
+```
+
+</Tab>
+</Tabs>
+
+
+## Initialize the Database
+
+Before you can start Boundary, the database must be initialized. To do this, run:
+
+```
+$ boundary database init -h
+...
+```
+
+From this command, you can see the flags available to skip the creation of auto-generated resources. These
+resources are useful when running in development environments but in a non development environment may not
+be desired. You can think of them as Boundary's default username and password, and if used in non dev workflows
+should be overidden or skipped to avoid undermining Boundary's security model.
+
+To initialize the Boundary database without generated resources:
+
+```
+$ boundary database init \
+  -skip-auth-method-creation \
+  -skip-host-resources-creation \
+  -skip-scopes-creation \
+  -skip-target-creation \ 
+  -config /etc/boundary.hcl
+```
+
+When you start Boundary, you will effectively have a blank sheet to work against. From here, it's required that
+you use the KMS recovery workflow described above to create at a minimum an auth method, a user, an account, and a 
+role with sufficient grants. Otherwise, you need to continue to use the recovery workflow for management. Though 
+continuing with the recovery workflow is not discouraged it's important to realize that this is effectively a 
```

Comment:
I think saying it's not discouraged is a little _too_ encouraging. Maybe just state the warning and that in some circumstances it may be desirable to continue to use the workflow, but it must be done with caution.